collectors/python.d.plugin/am2320/metadata.yaml

```diff
@@ -61,7 +116,7 @@ metrics:
   availability: []
   scopes:
   - name: global
-    description: ""
+    description: "These metrics refer to the entire monitored application."
```

Comment:
Can you add this snippet to the rest of the metrics? GitHub UI doesn't let me propose a change there.

Welcome to the world of forgetting stuff to copy paste in yamls 😄